modules/ROOT/pages/storage.adoc

```diff
@@ -128,3 +128,41 @@ storage:
 ----
 
 NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
+
+== Encrypted storage (LUKS)
+
+Here is an example to configure a LUKS device at `/var/lib/data`.
+
+[source.yaml]
+----
+variant: fcos
+version: 1.2.0-experimental
+storage:
+  luks:
+    - name: data
+      device: /dev/vdb
+  filesystems:
+    - path: /var/lib/data
+      device: /dev/disk/by-id/dm-name-data
+      format: xfs
+      label: DATA
+      with_mount_unit: true
+----
+
```

Comment:
Should we add some explanation test here to introduce the next example? (And e.g. link to Clevis.)